### DISCUSSION TITLE
Organisations should have their own RSS Feed import

### DISCUSSION THREAD

**Discussion by philnash** [forem/forem#15319] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
We have an organisation and one of our admins imports the Orgs RSS feed. Until recently the articles would arrive in the Org dashboard, but that broke and they started appearing in her dashboard. Fixing that assignment would be a temporary fix.

**Describe the solution you'd like**
An organisation should have an RSS feed URL, like any user.
The feed should be imported at the same cadence as regular user feeds.
Articles that are imported should be attached to the organisation by default.
Attaching the article to an author as well is tricky. We could:
* Attach it to no author (though I imagine that's against the current schema)
* Attach it to a default admin author (similar to what is happening now anyway)
* Try to ascertain which author it should be attached to based on certain RSS elements (perhaps not possible/in the standard?)

**Describe alternatives you've considered**
A user account that is solely the owner of the organisation and for all other purposes not a real account. This could then have credentials shared amongst a team so that people could log in to administer the organisation.

But that seems like a faff and really all things for the organisation should be able to be administered by an organisation admin.

**Comment by jessleenyc** [comment]
Some additional thoughts from @michael-tharrington here: https://github.com/thepracticaldev/dev.to/issues/4373

**Comment by philnash** [comment]
@michael-tharrington would love for this to be addressed at some point (and don't currently have the time to work on something myself). This has broken our workflow for the Twilio organisation right now and I'd love for it to get fixed.

**Comment by michael-tharrington** [comment]
@philnash you've outlined this feature in excellent detail. This particular issue has definitely been on my mind a lot recently — I'll make sure the team is aware this is a major pain point for Twilio.

**Comment by bnb** [comment]
Just want to add a +1 for this. It is currently a blocker for ramping up official Node.js usage of the platform for the Node.js org.

**Comment by dotNetkow** [comment]
The [Ionic team](https://dev.to/ionic) would love this! Since new posts currently appear under "Personal", not the Organization, only the person that originally set up the RSS feed in the first place can see them - making it hard to collaborate.

**Comment by simonaronsson** [comment]
The [k6](https://k6.io/) team would appreciate this as well. I'm struggling to find a way to easily collaborate on this right now.

As it seems like a lot of teams way larger than us have similar pain points: any chance this could be bumped up a bit in priority, @michael-tharrington ?

**Comment by michael-tharrington** [comment]
Ahh @simskij — I'm so sorry for missing your @mention here. I came back to this issue as there's another org admin who ran into some difficulties this feature would solve. I'll flag this one to the team again as I think this is an excellent idea!

**Comment by philnash** [comment]
I was just coming by to check on this and see there's been a bit of an update, which is good news!

I'd love to try to work on this, but I feel like getting a direction from the team as to how they would like this solved would be the best place to start. Also, it's been a while since I dug into the source code :D

**Comment by mstruve** [comment]
This is a user-facing feature so I am going to remove it from the EOY Project board since we really want to take this time to focus on internal tooling and/or code improvements and enhancements.

**Comment by ellativity** [comment]
Boosting this one, as it has been requested one again (this time by @ajonp - just connecting you with your request here, Alex!)

**Reply by codercatdev** [reply]
Thanks @ellativity, I would love to see multiple rss per account as well, unless I am just missing that feature

**Reply by michael-tharrington** [reply]
I just wrote this request up in its own discussion on behalf of another person who suggested it - https://github.com/forem/forem/discussions/17704

Figured I'd make it known in case you'd like to upvote!

**Comment by EisenbergEffect** [comment]
I'd really love to see this as well. The FAST team at Microsoft is very interested in syndicating our content to the new FAST organization in the webcomponents.dev community.

**Comment by rdegges** [comment]
Chiming in on behalf of [Snyk](https://snyk.io/), we'd love to see this as well. We're hoping to syndicate more of our technical articles to dev.to -- we love the platform!

**Comment by codercatdev** [comment]
I have pivoted and moved everything over to the API now. Adds a little more
control :D

On Mon, Aug 29, 2022 at 11:53 AM Randall Degges ***@***.***>
wrote:

> Chiming in on behalf of Snyk <https://snyk.io/>, we'd love to see this as
> well. We're hoping to syndicate more of our technical articles to dev.to
> -- we love the platform!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/forem/forem/discussions/15319#discussioncomment-3502664>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXVMKPC4YGPKWJRZ5HKTBTV3T2IHANCNFSM5KS2LKOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Thank You,
Alex Patterson Need to meet? Book a time: https://calendly.com/codingcatdev

🌎CodingCat.dev <https://codingcat.dev> || 🎥 YouTube
<https://link.ajonp.com/youtube> || 🐦 Twitter
<https://twitter.com/CodingCatDev> || 📷 Instagram
<https://www.instagram.com/codingcat.dev/>
